src/styl/responsive/tablet-portrait.styl

```diff
@@ -23,6 +23,9 @@ $filter-height = 63px
   ul
     justify-content: flex-end
 
+.social-icon
+  margin: 0 0 .8em 0
```

Comment:
I think one of these `0` is extra? On my local it looks proper with .8em left and right not just bottom?